testnets/doravotatestnet/chain.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "doravotatestnet",
```

Comment:
Will the Mainnet also be called Dora Vota? I thought I've seen somewhere a chain being called 'Dora', and assuming that is the mainnet counterpart to this chain, then I'd suggest the chain_name here be 'doratestnet' so that the mainnet can be called 'dora'. Otherwise, with this name, the mainnet's chain_name must be registered as 'doravota'

Reply:
Yes, the name of our chain is `Dora Vota` (the testnet is `Dora Vota Testnet`), so that's what I'm setting up here is `doravotatestnet`.